### ISSUE TITLE
Add ability to Blockstack for identity and authentication

### ISSUE THREAD

**Issue by engelgabriel** [RocketChat/Rocket.Chat#9392]
See https://blockstack.github.io/blockstack.js/index.html#authentication

**Comment by engelgabriel** [comment]
@timkinnane can we merge the fork that added this functionality?

**Comment by timkinnane** [comment]
@engelgabriel No. There's been no progress on this and it requires dedicated resources and ownership to remove blockers. I can be a part of that process, walk someone through, but I don't have the bandwidth to lead it right now in parallel with my other responsibilities.
